server/package.json

```
@@ -6,7 +6,7 @@
   "keywords": [],
   "author": "",
   "scripts": {
-    "start": "nodemon index.js",
+    "start": "NODE_ENV=development nodemon index.js",
```

you can't use ``` start ``` with nodemon because the deployment environment relies on it  
```suggestion
    "start": "NODE_ENV=production node index.js",
    "start:dev": "NODE_ENV=development nodemon index.js",
```